scripts/ci

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -e
 
-cd $(dirname $0)
```

Comment:
Using `$HOME` here will prevent us from building this repository locally, which probably isn't a good thing. I can't speak on how often (if ever) this repo is built locally, so I'll leave it up to the other reviewers to determine if this is acceptable or not - just want to point it out

Reply:
`dirname $0` resolves to `.` on these runners, which is why i had to change it.

Reply:
I would still say `$HOME` is not correct. Regardless of the frequency of builds, we need to figure out a way to resolve this without relying on `$HOME`. @mitulshah-suse when you run `make ci` locally, what does `$HOME` resolve to within the dapper container?